billing-tests/src/androidTest/java/com/mapbox/navigation/billing/test/BillingTests.kt

```diff
@@ -76,19 +66,14 @@ class BillingTests : BaseCoreNoCleanUpTest() {
         }
     }
 
-    @After
-    fun cleanUp() {
-        HttpServiceEventsObserver.unregister()
-    }
-
     @Test
     fun test30SecondsLongActiveGuidanceTrip() = sdkTest(TimeUnit.MINUTES.toMillis(1)) {
         val routes = requestRoutes(source, destination)
 
         mapboxNavigation.setNavigationRoutesAndWaitForUpdate(routes)
         mapboxNavigation.startTripSessionAndWaitForActiveGuidanceState()
 
-        delay(TimeUnit.SECONDS.toMillis(30))
+        delay(TimeUnit.SECONDS.toMillis(35))
```

Comment:
Why?

Reply:
Sessions <= 30 seconds are not billed.

Reply:
the test didn't pass with 30 second value, did it?

Reply:
We didn't have verification job before, it's introduced only in this PR.